src/psyclone/psyir/symbols/symbol.py

```diff
@@ -97,11 +96,11 @@ def __init__(self, container_symbol):
         # pylint: disable=import-outside-toplevel
         from psyclone.psyir.symbols import ContainerSymbol
 
-        super(GlobalInterface, self).__init__()
+        super(ImportInterface, self).__init__()
 
         if not isinstance(container_symbol, ContainerSymbol):
             raise TypeError(
-                "Global container_symbol parameter must be of type"
+                "Import container_symbol parameter must be of type"
```

Comment:
s/Import/ImportInterface/ will be clearer I think.

Reply:
Yes, done